features/buyers/accounts/edit.feature

```diff
@@ -16,8 +16,8 @@ Feature: Audience > Accounts > Edit
     Given they go to the buyer account edit page for "Pepe"
     When the form is submitted with:
       | Organization/Group Name | Pepito |
-    Then the current page is the buyer account page for "Pepito"
-    And should see a toast alert with text "Account successfully updated"
+    Then should see a toast alert with text "Account successfully updated"
+    And the current page is the buyer account page for "Pepito"
```

Comment:
The `the current page is` would wait, but `account page for "Pepito"` consults the DB, so the entry might not yet be there if the request has not yet been processed after clicking the button, so putting the UI step first to ensure the request is completed.

Reply:
I don't understand this. What's "the entry" here? So the problem is Capybara doesn't wait for the DB request to complete?

Reply:
Testing for the toast alert makes also sense in the scenario. I'm not sure we can make these tests fool proof. At the end UI tests are very hard to get right.